.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -59,3 +56,17 @@ If your plugin isn't production ready, please leave a comment stating whether yo
 
 - [ ] The plugin must set a `min_server_version` in the manifest.
 - [ ] The plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.
+
+**Documentation requirements (checked by a Technical Writer)**
+
+- [ ] The plugin must include detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md). Typical components of documentation include:
```

Comment:
Can we put the development guide as a separate linked .MD file?  Of the hundreds of people who read it, only a small fraction will ever custom code or contribute.  and ittakes up a large section.  I'd be happy to put a "How to Contribute" link higher up in the doc more prominently if it was linked out.  Otherwise, I say sink it to the bottom of the readme.

Reply:
Looking at this, I agree with you. It's too wordy. I believe we have this documented elsewhere but if not it certainly can be moved. I'll address this in a separate PR. Thank you!